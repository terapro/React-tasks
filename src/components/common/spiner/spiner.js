import React from 'react';

export const Spiner = () => (
  <img src='img/spiner.gif' className='spiner' />
);
