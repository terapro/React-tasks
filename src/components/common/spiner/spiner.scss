.spiner {
  width: 70px;
  display: block;
  margin: $elementPadding auto;

}