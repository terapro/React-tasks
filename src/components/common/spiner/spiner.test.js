import React from 'react';
import {Spiner} from 'src/components/common/spiner/spiner';
import {shallow} from 'enzyme';

describe('<Spiner />', () => {
  describe('Rendering', () => {
    it('should render the component', () => {
      const wrapper = shallow(<Spiner />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});