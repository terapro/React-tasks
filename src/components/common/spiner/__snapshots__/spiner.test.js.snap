// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spiner /> Rendering should render the component 1`] = `
<img
  className="spiner"
  src="img/spiner.gif"
/>
`;
